wmn-data.json

```diff
@@ -3629,6 +3641,18 @@
         "m_code" : 404,
         "known" : ["Djelizamay","Andysparkles"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "myportfolio",
+        "uri_check" : "https://{account}.myportfolio.com/work",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "digital artist, illustrator, game art",

```

Comment:
This is something that varies based upon the user account. For instance, https://bqwphotography.myportfolio.com does not have those data points.

Reply:
How about `class="page-title"`